.agent-os/instructions/core/import-ai-prompts.md

```
@@ -0,0 +1,304 @@
+Use GitHub CLI to retrieve all comments from the PR, focusing on CodeRabbit AI review comments.
+
+<github_cli_commands>
+  # Fetch PR comments with full body content
+  gh api repos/{owner}/{repo}/pulls/{number}/comments --paginate > pr_comments.json
+  
+  # Also fetch issue comments (some bots post here)
+  gh api repos/{owner}/{repo}/issues/{number}/comments --paginate > issue_comments.json
+</github_cli_commands>
+
+<comment_filtering>
+  - Filter for comments by 'coderabbitai' user
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)